pyqtgraph/parametertree/SystemSolver.py

```diff
@@ -52,16 +53,16 @@ class SystemSolver(object):
     1) The *defaultState* class attribute: This is a dict containing a 
        description of the variables in the system--their default values,
        data types, and the ways they can be constrained. The format is::
-       
+       ```
```

Comment:
`SystemSolver` isn't currently included in the docs but the two colons at the end of the line above put this indented block in a literal environment.